lib/datasource/conan/index.ts

```diff
@@ -0,0 +1,117 @@
+import { logger } from '../../logger';
+import { Http } from '../../util/http';
+import { ensureTrailingSlash } from '../../util/url';
+import * as loose from '../../versioning/loose';
+import type {
+  DigestConfig,
+  GetReleasesConfig,
+  Release,
+  ReleaseResult,
+} from '../types';
+
+export const id = 'conan';
+export const customRegistrySupport = true;
+export const defaultRegistryUrls = ['https://conan.bintray.com/'];
+export const defaultVersioning = loose.id;
+export const registryStrategy = 'merge';
+export const caching = false;
+
+const http = new Http(id);
```

Comment:
Can you write this as a class-based datasource? If you search for `extends Datasource` you'll find the ones I've written so far. If you do, you'll get error handling and caching for free 😄

Reply:
I'll give it a shot.  I got annoying behaviour with caching during testing a self hosted conan datasource, probably just something I misconfigured.

Reply:
Thanks! I hope that the existing datasources + tests are good examples, but if not, drop any questions below or on the Renovate slack channel.